transformato/analysis.py

```diff
@@ -96,15 +99,15 @@ def __init__(self, configuration: dict, structure_name: str):
         self.save_results_to_path: str = f"{self.configuration['system_dir']}/results/"
         self.traj_files = defaultdict(list)
 
-    def load_trajs(self, nr_of_max_snapshots: int = 300):
+    def load_trajs(self, nr_of_max_snapshots: int = 300, consecutive_runs: int = 0):
```

Comment:
I'd prefer calling this `multiple_runs`. While `consecutive` is also true in most cases, I don't think this is enforced.